packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -79,19 +76,13 @@ contract FundingRateApplier is Lockable {
      * @notice Constructs the FundingRateApplier contract. Called by child contracts.
      * @param _finderAddress Finder used to discover financial-product-related contracts.
      * @param _fundingRateIdentifier Unique identifier for DVM price feed ticker for child financial contract.
-     * @param _timerAddress Contract that stores the current time in a testing environment.
      * Must be set to 0x0 for production environments that use live time.
```

Comment:
if you can indeed remove the timer (and it works) then you should remove this comment as well ;)

Reply:
Oops nice catch!!

Reply:
Done.